src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -215,10 +226,13 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
         <div className={classes.groupHeader}>
           <DetailsRow
             {...props}
+            styles={customStyles}
             onClick={() => {
               if (!selectionDisabled) {
                 this.querySelected(props.item);
               }
+              this.resetCollapse = false;
+              this.setState({ selectedQuery: props.item.id })
```

Comment:
Passing the whole `props.item` into the `selectedQuery` instead of just the id helps get the category of the query and use it to set the expanded state of the category in the groups function

```suggestion
              this.setState({ selectedQuery: props.item})
```